modelzoo/DBMTL/README.md

```diff
@@ -0,0 +1,184 @@
+            └──────┘     │                  │     │         │    │          │    └─────────┘                                          
+                         │                  │     │         │    │          │
+                         │                  │     │         │    │          │
+            ┌──────┐     │                  │────►│         │───►│          │    
+            │      │     │                  │     └─────────┘    │          │    
+            │      │     │                  │                    │          │    
+Context     │      │────►│                  │                    │          │
+            │      │     │                  │                    └──────────┘ 
+            └──────┘     └──────────────────┘                                           
+                                                                                                    │_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _│
+                        Shared Embedding Layer                    Bottom Dnn   Specific Layer                                      Bayesian Layer
+```
+## Usage
+
+### Stand-alone Training
+1.  Please prepare the [data set](#prepare) first.
+
+2.  Create a docker image by DockerFile.   
+    Choose DockerFile corresponding to DeepRec(Pending) or Google tensorflow.
+    ```
+    docker build -t DeepRec_Model_Zoo_DBMTL_training:v1.0 .
+    ```
+
+3.  Run a docker container.
+    ```
+    docker run -it DeepRec_Model_Zoo_DBMTL_training:v1.0 /bin/bash
+    ```
+
+4.  Training.  
+    ```
+    cd /root/
+    python train.py
+    ```
+    Use argument `--bf16` to enable DeepRec BF16 in deep model.
+    ```
+    python train.py --bf16
+    ```
+    Use arguments to set up a custom configuation:
+    - `--data_location`: Full path of train & eval data, default is `./data`.
+    - `--output_dir`: Full path to output directory for logs and saved model, default is `./result`.
+    - `--checkpoint`: Full path to checkpoints input/output directory, default is `$(OUTPUT_DIR)/model_$(MODEL_NAME)_$(TIMESTAMPS)`
+    - `--steps`: Set the number of steps on train dataset. Default will be set to 10 epoch.
+    - `--batch_size`: Batch size to train. Default is 512.
+    - `--timeline`: Save steps of profile hooks to record timeline, zero to close, defualt to 0.
+    - `--save_steps`: Set the number of steps on saving checkpoints, zero to close. Default will be set to 0.
+    - `--keep_checkpoint_max`: Maximum number of recent checkpoint to keep. Default is 1.
+    - `--learning_rate`: Learning rate for network. Default is 0.1.
+    - `--bf16`: Enable DeepRec BF16 feature in DeepRec. Use FP32 by default.
+    - `--no_eval`: Do not evaluate trained model by eval dataset.
+    - `--protocol`: Set the protocol("grpc", "grpc++", "star_server") used when starting server in distributed training. Default to grpc. 
```

Comment:
Add `inter`, `intra`, `input_layer_partitioner` and `dense_layer_partitioner` information to README

Reply:
Changed